lib/tasks/import.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+namespace :import do
```

Comment:
I think i'd like to nest the namespace to `:mahonia` and `:import`, but will probably aim to do that in the follow-up that adds the generic task.

Reply:
Oh, good call. Yes, thank you.